src/containers/header/Header.jsx

```diff
@@ -45,7 +45,7 @@ function Header() {
                       <a href="#team">Team</a>
                   </p>
                     <p>
-                        <a href="https://files.lighthouse.storage/">Upload</a>
+                        <a href="https://files.lighthouse.storage/">Files DApp</a>
```

Comment:
Files dapp here is not visual, seems like a regular button. Either create a section for it or change visual